app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -54,11 +54,13 @@ fun Greeting(name: String) {
         Button(onClick = { /*TODO*/ })
         {Text(text = "Task Timed")}
 
+        Button(onClick = { /*TODO*/ })
+        {Text(text = "Start Timer")}
```

Comment:
Why is the button moved here?  Is this changing design, or does it have a significant impact on technical debt?